packages/gravity-ui-web/src/sass/05-components/02-molecules/00-navigation/_00-nav-menu.scss

```diff
@@ -5,6 +5,7 @@
   > ul {
     margin: 0 (-$grav-sp-s);
     padding: 0;
+    padding-right: $grav-sp-l;
```

Comment:
I think this padding's in the wrong place. We need to expand the gap between adjacent nav-links, but this jjust creates some space on the right of the list itself.

I suggest the following:
```scss
.grav-c-nav-menu {
  // ...

  > ul {
   // ...

    // Remove this one:
    // padding-right: $grav-sp-l;

    // ...
  }

  li {
    // ...
  }

  // Add this:
  // When we switch to a horizontal layout, we want
  // to add a left margin on all <li>s, except the first
  // (i.e. left-most) one:
  @media (min-width: $trigger-breakpoint) {
    li + li {
      margin-left: $grav-sp-l;
    }
  }

  li:not(#{&}__external-link) + li#{&}__external-link {
    // Change this:
    // This is to create a larger gap between the normal links
    // and the first of the (optional) external links. The old "small"
    // gap looks a bit too small, so I suggest increasing it to "medium":
    margin-top: $grav-sp-m;

    @media (min-width: $trigger-breakpoint) {
      margin-top: 0;

      // Change this:
      // Same thing as above, but in horizontal layout we need to
      // add a bigger kleft margin. Since we've added a "large" margin
      // between all normal links, we need to increase this one to maintain
      // a visibly larger gap to the external links. I suggest "XL":
      margin-left: $grav-sp-xl;
    }
  }
}
```

Reply:
Sorry that I made that mistake. Thanks for your help getting this finished up.